src/main/java/org/jenkinsci/plugins/ghprb/GhprbBuilds.java

```diff
@@ -169,15 +171,19 @@ public void onCompleted(AbstractBuild<?, ?> build, TaskListener listener) {
         // remove the BuildData action that we may have added earlier to avoid
         // having two of them, and because the one we added isn't correct
         // @see GhprbTrigger
-        BuildData fakeOne = null;
         for (BuildData data : build.getActions(BuildData.class)) {
             if (data.getLastBuiltRevision() != null && !data.getLastBuiltRevision().getSha1String().equals(c.getCommit())) {
-                fakeOne = data;
+                build.getActions().remove(data);
                 break;
             }
         }
-        if (fakeOne != null) {
-            build.getActions().remove(fakeOne);
+        
+
+        if (build.getResult() == Result.ABORTED) {
```

Comment:
So this avoids changing the github status if the build was aborted by the Ghprb?